Fixing package.json for windows

karanvirsb opened the pull request:
Added fix for windows as core package json would give issues due to the cp command not existing on windows. Along with this change added new scripts to packages/mirrorful and removed removeFileOrDirectory.ts as it is no longer needed.

alexdanilowicz commented:
Oh very interesting, have not heard of https://www.npmjs.com/package/run-script-os before.

I think `run-script-os` should be a dev dependency tho, and installed with `npm install --save-dev run-script-os`.

Looks like you have it installed a regular dependency. Could you fix that @karanvirsb  ? Thanks for opening this!

karanvirsb commented:
> Oh very interesting, have not heard of https://www.npmjs.com/package/run-script-os before.
> 
> I think `run-script-os` should be a dev dependency tho, and installed with `yard add -D run-script-os`.
> 
> Looks like you have it installed a regular dependency. Could you fix that @karanvirsb ? Thanks for opening this!

I have fixed it and moved it into the dev dependencies!

alexdanilowicz commented:
Great, we'll get this merged in today!
